RELEASENOTES.md

```diff
@@ -0,0 +1,118 @@
+# Release Notes
+
+This document highlights what is new in major versions of Kirby. See the [Changelog](https://github.com/kirbydesign/designsystem/blob/main/CHANGELOG.md) and the [Migration Guide](https://github.com/kirbydesign/designsystem/blob/main/MIGRATION.md) for more details.
+
+- [Version 7](#version-7) August 2022
+
+## Version 7
+
+Version 7 is mostly about improving the user experience for other devices than mobile/touch-based devices. In other words, making Kirby components more _desktop ready_.
+
+### What's new?
+
+- [Button attention level 4 has been deprecated](#button-attention-level-4-has-been-deprecated)
+- [Interaction States](#interaction-states)
+- Charts
+  - New _Stock Chart_ component
+  - _Highcharts_ dependency removed
+  - General refactoring
+- Refactor moving of modal elements to solve multiple issues
```

Comment:
Could we be a bit more specific on what these multiple solved issues are?

Reply:
Yes. I'm not sure how much detail the POs need, though.

Reply:
Yeah you are right. Maybe we should just reference the PR or the issues solved?

Reply:
Ok, I added the titles of the individual issues plus a link to the PR